dev_docs/getting_started/hello_world_plugin.mdx

```diff
@@ -0,0 +1,144 @@
+---
+id: kibHelloWorldApp
+slug: /kibana-dev-docs/hello-world-app
+title: Hello World
+summary: Build a very basic plugin that registers an application that says "Hello World!".
+date: 2021-08-03
+tags: ['kibana', 'dev', 'contributor', 'tutorials']
+---
+
+This tutorial walks you through two ways to create a plugin that registers an application that says "Hello World!".
+
+You can view the tested example plugin at [examples/hello_world](https://github.com/elastic/kibana/tree/master/examples/hello_world).
+
+## 1. Set up your development environment
+
+Read through <DocLink id="kibDevTutorialSetupDevEnv" text="these instructions"/> to get your development environment set up.
+
+## 2. Option 1 - Write it manually
+
+This is a good option if you want to understand the bare minimum needed to register a "Hello world" application. The example plugin is based off of this option.
+
+1. Create your plugin folder. Start off in the `kibana` folder.
+
+```
+> cd examples
+> mkdir hello_world
+> cd hello_world
+```
+
+2. Create the kibana.json manifest file.
+
+```
+> echo '{
+  "id": "helloWorld",
+  "version": "1.0.0",
+  "kibanaVersion": "kibana",
+  "ui": true
+}' > kibana.json
+```
+
+3. Create a `tsconfig.json` file.
+
+```
+> echo '{
```

Comment:
should we rather just provide contents of the file (which can be syntax higlighted then) ?

Reply:
sgtm, done